rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -949,4 +958,160 @@ impl SDKClient {
         instructions.push(post_only_ix);
         Ok(instructions)
     }
+
+    // Create a limit order instruction from a LimitOrderTemplate
+    pub async fn get_limit_order_ix_from_template(
```

Comment:
wdyt of grabbing the market metadata upstream and passing it in to this function? (or passing in base_Mint and quote_mint)?

feels weird for this function to be async

Reply:
I think this is a good call. Will also reduce the number of network calls since we use the same metadata for both bid and ask templates.